app/src/common/middleware/assets.mjs

```diff
@@ -8,7 +8,7 @@ const router = express.Router()
 /**
  * Add middleware to serve static assets
  */
-router.use('/assets', express.static(join(paths.app, 'dist/assets')))
+router.use('/assets', express.static(join(paths.src, 'govuk/assets')))
```

Comment:
We'll swap this with `node_modules/govuk-frontend` once we've changed our package structure